src/scss/components/_authentication-errors.scss

```diff
@@ -0,0 +1,3 @@
+.openforms-authentication-errors {
+  margin: 1rem 0;
```

Comment:
please don't use shorthands so that we can use styles that are writing direction independent:

```css
margin-block-end: 1rem;
margin-block-start: 1rem;
margin-inline-end: 0;
margin-inline-start: 0;
```
also - does this need to be margins? Blocks should not specify margins, but paddings to create vertical spacing so that the components can be rearranged properly.

Perhaps a better solution to create vertical spacing would also be to apply a flexbox layout to the container with appropriate `gap` values set?

Reply:
I don't really know why they were margins :thinking: I tried looking on our test environment and the one for DH, but couldn't get this alert to be rendered in order to check it. So I kept as close as possible to what we had before :S